src/parser/csql_grammar.y

```diff
@@ -26,6 +26,10 @@
 #include "json_table_def.h"
 #include "parser.h"
 
+/* No longer supports pt_type_nchar; Keep grammar replaced with pt_type_char type.*/
+#define PT_TYPE_VARNCHAR   PT_TYPE_VARCHAR
+#define PT_TYPE_NCHAR      PT_TYPE_CHAR
```

Comment:
It seems not the proper place... How about just setting the same value for PT_TYPE_VARNCHAR and PT_TYPE_NCHAR in `pt_type_enum`? 
```
enum pt_type_enum
{
...
  PT_TYPE_CHAR,
  PT_TYPE_VARCHAR,
  PT_TYPE_NCHAR = PT_TYPE_CHAR,
  PT_TYPE_VARNCHAR = PT_TYPE_VARCHAR,
...
}
```